templates/webapi/NewJob/create.html.ep

```diff
@@ -0,0 +1,122 @@
+% layout 'bootstrap';
+% content_for 'head' => begin
+  %= asset 'ace.js'
+  %= asset 'ace.css'
+% end
+
+% title 'Run a Job';
+
+<div>
+  <h2><%= title %></h2>
+  
+  %= include 'layouts/info'
+  <p><b>Hint:</b> Experimental form to trigger a test E.g. see <%= link_to 'this
+								      link' => url_for('not_found') %>.
+  </p>
+  <div class="container-md">
+  <form class="row g-3" onsubmit="runExample(this);" data-post-url="/api/v1/isos">
```

Comment:
We should not hardcode paths like `/api/v1/isos`. Use the Mojolicious helper `url_for` instead.